target/linux/ath79/dts/ar9344_mikrotik_sxt5n.dtsi

```diff
@@ -0,0 +1,182 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+/dts-v1/;
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+#include "ar9344.dtsi"
+
+/ {
+	compatible = "mikrotik,sxt5n", "qca,ar9344";
+	model = "MikroTik SXT5N platform";
+
+	aliases {
+		label-mac-device = &eth1;
+		led-boot = &led_power;
+		led-failsafe = &led_power;
+		led-running = &led_power;
+		led-upgrade = &led_power;
+		serial0 = &uart;
+	};
+
+	chosen {
+		bootargs = "console=ttyS0,115200n8";
+	};
+
+	leds: leds {
+		compatible = "gpio-leds";
+
+		led_power: power {
+			label = "sxt5n:green:power";
+			gpios = <&gpio 11 GPIO_ACTIVE_HIGH>;
+		};
+
+		wlan {
+			label = "sxt5n:green:wlan";
+			gpios = <&gpio 13 GPIO_ACTIVE_LOW>;
+			linux,default-trigger = "phy0tpt";
+		};
```

Comment:
I don't see any LED setup in ar71xx, so is this assignment of the 5 "Wifi LEDs" based on vendor or did you invent it?

Reply:
It is my invention but, incidentally, RouterOS incidentally sets the LEDs like this:

![image](https://user-images.githubusercontent.com/6009603/85286202-79ce2780-b492-11ea-9137-f11c0a1e2759.png)

By the way, how would you interpret this:

![image](https://user-images.githubusercontent.com/6009603/85286424-ec3f0780-b492-11ea-8cee-1893fc7f4838.png)

1xWLAN + 4xRSSI, or 5xRSSI?

Reply:
Note: having 1xWLAN + 4xRSSI means one LED is indicating if the interface is up/down and if there's traffic on it, plus the other 4 indicating link quality. Having 5xRSSI only indicates (more precisely) link quality.

Reply:
In routerOS it looks like its meant to be interpreted as 5x RSSI. However I don't see the point, and it doesnt mean we have to do that